src/tcgwars/logic/impl/gen8/DarknessAblaze.groovy

```diff
@@ -0,0 +1,3394 @@
+          actionA {
+          }
+        }
+        move "Rock Throw", {
+          text "30 damage. "
+          energyCost F, C
+          attackRequirement {}
+          onAttack {
+            damage 30
+          }
+        }
+      };
+      case HIPPOPOTAS_96:
+      return basic (this, hp:HP100, type:F, retreatCost:4) {
+        weakness G
+        move "Tackle", {
+          text "10 damage. "
+          energyCost C
+          attackRequirement {}
+          onAttack {
+            damage 10
+          }
+        }
+        move "Double Stomp", {
+          text "70+ damage. Flip 2 coins. This attack does 30 more damage times the number of heads."
+          energyCost F, F, F, C
+          attackRequirement {}
+          onAttack {
+            damage 70
+          }
+        }
+      };
+      case HIPPOWDON_97:
+      return evolution (this, from:"Hippopotas", hp:HP150, type:F, retreatCost:4) {
+        weakness G
+        move "Triple Smash", {
+          text "80x damage. Flip 3 coins. This attack does 80 damage for each heads."
+          energyCost F, F, C
+          attackRequirement {}
+          onAttack {
+            damage 80
+          }
+        }
+        move "Land Crush", {
+          text "150 damage. "
+          energyCost F, F, F, C
+          attackRequirement {}
+          onAttack {
+            damage 150
+          }
+        }
+      };
+      case RHYPERIOR_V_98:
+      return basic (this, hp:HP230, type:F, retreatCost:4) {
+        weakness G
+        move "Drill Run", {
+          text "80 damage. Discard 1 Energy from your opponent’s Active Pokémon."
+          energyCost F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 80
+          }
+        }
+        move "Heavy Rock Cannon", {
+          text "210 damage. This Pokémon can’t use this attack during your next turn."
+          energyCost F, C, C, C
+          attackRequirement {}
+          onAttack {
+            damage 210
+          }
+        }
+      };
+      case DIGGERSBY_99:
+      return evolution (this, from:"Bunnelby", hp:HP130, type:F, retreatCost:3) {
+        weakness G
+        move "Hammer In", {
+          text "90 damage. "
+          energyCost F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 90
+          }
+        }
+        move "Land Crush", {
+          text "140 damage. "
+          energyCost F, F, C, C
+          attackRequirement {}
+          onAttack {
+            damage 140
+          }
+        }
+      };
+      case ZYGARDE_100:
+      return basic (this, hp:HP140, type:F, retreatCost:3) {
+        weakness G
+        move "Bite", {
+          text "30 damage. "
+          energyCost C
+          attackRequirement {}
+          onAttack {
+            damage 30
+          }
+        }
+        move "Power Blast", {
+          text "130 damage. Discard an Energy attached to this Pokémon."
```

Comment:
Should say "Discard a [F] Energy attached to this Pokémon."

![](https://www.pokebeach.com/news/0320/038015_P_JIGARUDE.jpg)